bin/compile

```diff
@@ -15,13 +15,15 @@ if [[ -z "${PROCFILE}" ]]; then
     exit 1
 fi
 
-cp "${BUILD_DIR}/${PROCFILE}" "${BUILD_DIR}/Procfile"
+PROCFILE_BUILDED="${BUILD_DIR}/${PROCFILE}"
 
-if ! [ $? ]; then
+if [ ! -f "${PROCFILE_BUILDED}" ]; then
     echo "FAILED to copy a Procfile" | indent
+    echo "The file ${PROCFILE_BUILDED} doesn't not exists" | indent
     exit 1
 fi
 
+cp "${PROCFILE_BUILDED}" "${BUILD_DIR}/Procfile"
```

Comment:
this could still fail and we wouldn't be handling this

Reply:
There is no reason to fail, the script only will try to copy the file if the source file exists (checked by `-f`) and the build dir as well.

Reply:
https://github.com/heroku/heroku-buildpack-multi-procfile/pull/24/commits/50ee018d331fbe4dfb0e3b33d55004f5f581fbe3